docs/src/rules/no-restricted-imports.md

```diff
@@ -109,6 +109,18 @@ Pattern matches can also be configured to be case-sensitive:
 }]
 ```
 
+Pattern matches can restrict specific import names only, similar to the `paths` option:
+
+```json
+"no-restricted-imports": ["error", {
+    "patterns": [{
+      "group": ["utils/*"],
+      "importNames": ["isEmpty"],
+      "message": "Use 'isEmpty' from lodash instead."
+    }]
+}]
+```
```

Comment:
Can you also add examples (one with incorrect code, and one with correct code) in the Examples section?